### DISCUSSION TITLE
Lightning data sourcing question

### DISCUSSION THREAD

**Discussion by brlkid** [weather-gov/api#714] [Q&A]
Hello helpful community,
Some members, especially the experienced ones, may think this question is stupid especially if there is documentation about it. I am blind, so I have a hard time reading some of the published materials on the Internet and some of the websites are a bit broken to my screen reader.
With that being said, is there any method to gather lightning strike information? I know there are satellites that collect this data and I have heard that there are maps which can be downloaded live. How can I go about this and what or where can I gather the information About the service?
Thank you so much for your help and I look forward to hearing more information soon.

Thanks,
Derrick C. Day

**Comment by dwhitemv25** [comment]
What sort of data are you looking for? Lists of detections? Maps?

There's no lightning data in the NWS API, but the Blitzortung project (www.blitzortung.org) provides an open-source lightning database. Their terms of use are fairly specific, so review those and make sure they are appropriate for your project. 

Data from the GLM instrument on GOES-16 is available through CLASS in a shopping cart-like format (not appropriate for automated fetching, but fine for research). The files are in NetCDF format. More information available at https://ghrc.nsstc.nasa.gov/lightning/overview_glm.html

Lightning detection data is also available from commercial vendors.

**Comment by brlkid** [comment]
Thank you very much for the resources and guidance. Just to make sure I'm understanding correctly, there is no way to access the information pertaining to live lightning strikes from the NWS? I would like to alert people when a strike has occurred within a specific radius of their location, and I was trying to offer the service to blind people for free because of the safety implications that come with not being able to see the lightning flashes. Commercial options definitely cost money; most of which cost a substantial amount of money. I am just trying to find the most economical option and going to the source is often the best option. Thank you for your help, and I look forward to hearing from you very soon.

Thanks so much,
Derrick C. Day
________________________________

**Comment by dwhitemv25** [comment]
I have not found any National Weather Service source for lightning data. I'm pretty sure NWS is getting data from Viasala for their own use. 

I found that GOES GLM data is being sent to S3 and download assistance for that data can be found at https://lightning.umd.edu/glm/, on the DIY tab. The data is presented in NetCDF files in a gridded format so some processing is necessary to georeference it to a user location. Also the GLM resolution is fairly coarse.

Did you look into the Blitzortung data? 

Finally, free mobile weather apps commonly have "lightning near you" alerts. I know that my current favorite radar app does. Do those not meet your needs?

**Reply by sullynole** [reply]
Correct, NLDN and ENTLN are not operated by NWS. The satellite lightning products from GLM are a bit different than those options.

**Comment by brlkid** [comment]
OK, thank you very much for the information. I will look into my options and determine what works best for my project.

Thanks so much,
Derrick C. Day
________________________________
